src/components/Editor/EditorControlPane/Widget.js

```diff
@@ -84,14 +91,9 @@ export default class Widget extends Component {
     this.props.onValidate(errors);
   };
 
-  validatePresence(field, value) {
+  validatePresence = (field, value) => {
```

Comment:
Not certain we want to convert these to lexically scoped (fat arrow) functions, let's leave it as is to be safe. This applies to `validatePattern` as well.